components/compliance-service/ingest/server/compliance.go

```diff
@@ -167,7 +169,11 @@ func (s *ComplianceIngestServer) ProcessComplianceReport(ctx context.Context, in
 		SourceRegion:    in.SourceRegion,
 		SourceAccountId: in.SourceAccountId,
 		Tags:            in.Tags,
-	})
+		ScanData: &nodes.LastContactData{
+			Id:      in.ReportUuid,
+			EndTime: endTimeTimestamp,
```

Comment:
This change will make it so you don't have to pass the `profiles` to the `sendNodeInfoToManager` function. 
```suggestion
			EndTime: endTimeTimestamp,
			Status: getReportStatus(in.Profiles),
```

Reply:
ah, good point!
i had left that off to be done in the function b/c i (incorrectly) assumed that it was being done async.  but now i just noticed that that we check the error. 
I wonder if that function should be executed via go routine instead, to avoid slowing down the ingestion process?

Reply:
I would say not to run it in a separate go routine at this point. I would leave that for later when we look to increase the speed of ingestion.